src/lib/pages/backoffice/DocumentRequest/DocumentRequestEditor/DocumentRequestEditor.js

```diff
@@ -54,10 +58,19 @@ export class DocumentRequestEditor extends Component {
       },
     } = this.props;
 
-    const isEditing = !!documentRequestPid;
-    return isEditing
-      ? await documentRequestApi.update(documentRequestPid, formData)
-      : await documentRequestApi.create(formData);
+    try {
+      return this.userIsEditing
+        ? await documentRequestApi.update(documentRequestPid, formData)
+        : await documentRequestApi.create(formData);
+    } catch (error) {
+      this.setState({
```

Comment:
I think that this should not be handled here. This same pattern is use in all the forms (try to look for `.create(formData)` in the code).
You should try to change the wrapper cmp `RJSForm.js` in `onError` function instead so that it will be applied globally.

Reply:
ah ok! I had the more risk averse thought of not trying to change a general component but if you say so 😎